packages/FormElement/src/components/Label/Label.js

```diff
@@ -85,6 +88,7 @@ const defaultProps = {
   isDisabled: null,
   isVisuallyHidden: false,
   helpAlign: Popover.types.align.TOP,
+  zIndex: null,
```

Comment:
In this case, since this prop is only used for the "Help" popover, I would recommend we follow the convention that's already in place and prefix with `help` (ie `helpZIndex`).